content/800-data-platform/400-platform-console/50-support.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Support'
+metaTitle: 'Platform console: Support'
+metaDescription: ''
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Please reach out [here](https://pris.ly/cloud-projects-feedback) if you have any questions or concerns about Platfrom Console or Prisma Data Platform.
```

Comment:
```suggestion
Your feedback is invaluable, and we encourage you to share your experiences with us on Discord.
```

I would suggest the above if we need an interim solution until we can incorporate the rest of the support information.